sql/updates/world/master/9999_99_99_19344_world.sql

```diff
@@ -0,0 +1,7 @@
+DELETE FROM `spell_areatrigger` WHERE (`SpellMiscId`=5420 AND `AreaTriggerId`=10133);
+INSERT INTO `spell_areatrigger` (`SpellMiscId`, `AreaTriggerId`, `MoveCurveId`, `ScaleCurveId`, `MorphCurveId`, `FacingCurveId`, `DecalPropertiesId`, `TimeToTarget`, `TimeToTargetScale`, `VerifiedBuild`) VALUES
+(5420, 10133, 0, 0, 0, 0, 0, 0, 7177, 23420); -- SpellId : 5740
+
+DELETE FROM `spell_script_names` WHERE `spell_id`=5740;
```

Comment:
Please delete by `ScriptName` instead of `spell_id` to remove any existing spell script names using a different spell ID.